recipe/meta.yaml

```diff
@@ -1,27 +1,24 @@
-{% set name = "prompt_toolkit" %}
-{% set version = "2.0.10" %}
-{% set sha256 = "f15af68f66e664eaa559d4ac8a928111eebd5feda0c11738b5998045224829db" %}
+{% set version = "3.0.0" %}
 
 package:
-  name: {{ name|lower }}
+  name: prompt_toolkit
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/p/prompt_toolkit/prompt_toolkit-{{ version }}.tar.gz
+  sha256: cf1fc1500e9818973c4f92b6015bece88253f0aa599dd61d3fd375c5205cecbb
 
 build:
-  noarch: python
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
 
 requirements:
   host:
+    - python >=3.6
```

Comment:
@isuruf I was going to ask you about this but I guess it was merged already. This will be a lint in the next conda-smithy, right? How are we supposed to do ``noarch`` that skips a python?

Reply:
Those lints are only if `noarch: python` is not set. https://github.com/conda-forge/conda-smithy/pull/1193/files#diff-75c89ae9a2c5841bf730ed3474cc2987R455

Reply:
Ah. I thought that you would get latest python here too. So in this case it will be built with "lowest" 3 available. Thanks!

Reply:
You would get the latest python here, but noarch: python doesn't care.